packages/xo-server/src/_handleBackupLog.mjs

```diff
@@ -6,8 +33,44 @@ function forwardResult(log) {
 }
 
 // it records logs generated by `@xen-orchestra/backups/Task#run`
-export const handleBackupLog = (log, { logger, localTaskIds, rootTaskId, runJobId = rootTaskId, handleRootTaskId }) => {
-  const { event, message, taskId } = log
+export const handleBackupLog = (
+  app,
+  jobName = undefined,
+  log,
+  { logger, localTaskIds, rootTaskId, runJobId = rootTaskId, handleRootTaskId }
+) => {
+  const { event, message, parentId, taskId } = log
+
+  if (jobName) {
+    if (event === 'start') {
+      if (log.data?.type === 'VM') {
+        vmBackupInfo.set(taskId, {
+          type: 'VM',
```

Comment:
I think we can improve (simplify) even further the code by prefixing the task ids by their type in the map:
```suggestion
        vmBackupInfo.set('vm-' + taskId, {
```